### DISCUSSION TITLE
Starstruck

### DISCUSSION THREAD

**Discussion by bingyun123** [Schweinepriester/github-profile-achievements#427] [Q&A] [answered]
Hello world.

**Comment by xLevitate** [comment]
Done https://github.com/xLevitate/neptune

**Reply by drk1rd** [reply]
Done!
https://github.com/drk1rd/drk1rd

**Reply by callqh** [reply]
Done!
https://github.com/liuqh0609/note

**Reply by aarongeo1** [reply]
@drk1rd @liuqh0609 done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by ArielMAJ** [reply]
@xLevitate I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by drk1rd** [comment, marked as answer]
Done!
https://github.com/drk1rd/drk1rd

**Comment by callqh** [comment]
Done!
https://github.com/liuqh0609/note

**Comment by HHNebula** [comment]
Done! Do it for me, too
https://github.com/HHNebula/HHNebula

**Reply by aarongeo1** [reply]
@HHNebula done https://github.com/aarongeo1/NUM-CLASSIFY

**Reply by HHNebula** [reply]
@aarongeo1 Done!
